rubert/benches/mbert.rs

```diff
@@ -48,10 +49,10 @@ macro_rules! bench_tract {
 fn bench_onnx(
     manager: &mut Criterion,
     name: &str,
-    vocab: impl AsRef<Path>,
-    model: impl AsRef<Path>,
+    vocab: Result<impl AsRef<Path>>,
+    model: Result<impl AsRef<Path>>,
 ) {
-    let tokenizer = TokenizerBuilder::from_file(vocab)
+    let tokenizer = TokenizerBuilder::from_file(vocab.unwrap())
```

Comment:
The model path resolving is test specific maybe we want to move the unwrap into the `fn vocab()`,`fn model()` functions (they then using `resolve***.unwrap()`).

But I'm not sure about this.

But we never do not unwrap it.

Reply:
although we never not unwrap them, i'd prefer the utils crate to provide non-panicking/error handling if downstream crates may want to use it.